test/recipes/80-test_ssl_old.t

```diff
@@ -359,7 +359,7 @@ sub testssl {
 
       SKIP: {
 	  skip "SSLv3 is not supported by this OpenSSL build", 4
-	      if disabled("ssl3");
+	      if disabled("ssl3") || $provider eq "fips";
```

Comment:
A cosmetic issue - perhaps there should be a separate skip statement? "SSLv3 is not supported by fips provider" ?

Reply:
Good point. Fixed.